airbyte-integrations/connectors/source-shopify/source_shopify/schemas/products.json

```diff
@@ -268,6 +268,25 @@
     },
     "shop_url": {
       "type": ["null", "string"]
+    },
+    "is_deleted": {
+      "type": ["null", "boolean"]
+    },
+    "deleted_at": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "subject_type": {
+      "type": ["null", "string"]
+    },
+    "message": {
+      "type": ["null", "string"]
+    },
+    "author": {
```

Comment:
these fields (subject_type/message/author/description) seem specific to the `events` endpoint and don't tell the user anything about the `products` endpoint. Shouldn't they not be in the `products` stream? (similar question for every other stream). 

I think the only field we need to add here is `deleted_at`

Reply:
I've left the next fields:

```
{
    "id": event["subject_id"], # the id of the record (UUID)
    self.cursor_field: event["created_at"], # when the record was removed
    "updated_at": event["created_at"], # the regular stream cursor field as duplicate of the `deleted_at` basically (mandatory for CAT tests)
    "message": event["message"], # user-friendly message what was removed.
    "description": event["description"], # more info about  what was  removed,
    "shop_url": event["shop_url"], # the shop URL to identify the origin of the removed record.
}